ui/src/app/pages/data/concept/concept-table.tsx

```diff
@@ -107,23 +158,24 @@ export class SynonymsObject extends React.Component<{}, SynonymsObjectState> {
 }
 
 interface Props {
-  concepts: Concept[];
+  concepts: any[];
```

Comment:
Did we change this to any because it can also be PM?

Reply:
Actually changed so surveys could be included. We used to use a separate component but now we use the same one for all domains